app/src/main/java/com/mstudio/superheroarch/presentation/DetailsViewModel.kt

```diff
@@ -23,18 +23,25 @@ class DetailsViewModel(
 
     private suspend fun fetchFirstEpisodeDetails(episodeUrl: String) {
         try {
-            val episodeResult = getEpisodeDetailsUseCase(episodeUrl)
-            episodeResult.getOrNull()?.let {
-                view.displayFirstEpisodeDetails(it)
+            val result = getEpisodeAndDetailsUseCase(episodeUrl)
+            if (result.isSuccess) {
```

Comment:
unsae result.onSuccess{}.onFailure{}

Reply:
Solved in next commit¿?

Reply:
nope

Reply:
I think that now is solved in the next commit.